docs/basics/setting-up-the-client.md

```diff
@@ -0,0 +1,79 @@
+---
+title: setting-up-the-client
+order: 0
+---
+
+## Installation
+
+Installing `urql` is as quick as you'd expect. Firstly, install it
+with your package manager of choice, this installation is specific for React:
```

Comment:
Nit:

Firstly, install it with your package manager of choice. Note: this installation is specific to React.